prompt_toolkit/contrib/shortcuts.py

```diff
@@ -190,7 +190,7 @@ def get_input(message='',
 
     # Read input and return it.
     on_abort = AbortAction.RAISE_EXCEPTION if raise_exception_on_abort else AbortAction.RETURN_NONE
-    document = cli.read_input(on_abort=on_abort, on_exit=AbortAction.IGNORE)
+    document = cli.read_input(on_abort=on_abort, on_exit=AbortAction.RETURN_NONE)
```

Comment:
I will have a second look at this tomorrow. I'm not sure what the best behaviour is, actually.
If we look at what ``raw_input`` does for ``Ctrl-D`` and ``Ctrl-C``. it either throws ``KeyboardInterrupt`` or ``EOFError``.
Maybe not the most useful defaults, but somehow, I think if no parameters are given to ``get_input``, that it would be good if it behaves exactly as ``raw_input``.

What do you think?

(By the way: thank for the of the pull request! It looks good.)